website/src/main/tut/docs/rules/OrphanImplicits.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: docs
+title: DisableUnless
+---
+
+# OrphanImplicits
+
+_Since 0.6.0_
+
+This rule bans definitions of implicits F[G] unless they are on the companion of the F or the G.
+
+# Example
+
+```scala
+  trait Foo
+  trait Bar[T]
+
+  object Foo {
+    implicit val foo: Foo = ??? // ok
+    implicit val listFoo: List[Foo] = ??? // ok
+  }
+
+  object Bar {
+    implicit val foo: Foo = ??? // error
```

Comment:
Why is this an error? `Foo` is not of shape `F[G]`

Reply:
It's not a violation of

> This rule bans definitions of implicits F[G] unless they are on the companion of the F or the G.

Reply:
there is a move to avoid implicits that are not of shape `F[_]` in scalaz, so I'd be happy with this as an error!

Reply:
(but it's not an orphan problem... so this should not error under this rule)

Reply:
But is this an "orphan implicit error" or something else? I'll leave the final decision to you, just wanted to point out that the tests were not in sync with the documentation description.

Reply:
This is not an orphan implicit

Reply:
@fommil what about F[A, B] form? Is it should be ignored or not?

Reply:
`F[A, B]` can look like `F[_]` if one of the types is fixed. If you can test if this implicit is defined on `F`, `A` or `B` that would be good.

Reply:
Is this example up-to-date?